iNaturalist/src/main/java/org/inaturalist/android/MessageAdapter.java

```diff
@@ -85,7 +85,7 @@ public Bitmap onPreSetBitmap(ImageView imageView, Bitmap loadedBitmap, String ur
         Timestamp ts = message.getTimestamp("updated_at");
         Calendar cal = Calendar.getInstance(Locale.ENGLISH);
         cal.setTimeInMillis(ts.getTime());
-        holder.date.setText(DateFormat.format("MMMM d, yyyy hh:mma", cal));
+        holder.date.setText(DateFormat.format(mApp.isLayoutRTL() ? "d MMMM, yyyy HH:mm" : "MMMM d, yyyy hh:mma", cal));
```

Comment:
If this is just text, I think we should just handle this by globalizing the format string.